packages/next/client/use-intersection.tsx

```diff
@@ -16,14 +23,15 @@ type Observer = {
 const hasIntersectionObserver = typeof IntersectionObserver !== 'undefined'
 
 export function useIntersection<T extends Element>({
+  rootEl,
   rootMargin,
   disabled,
 }: UseIntersection): [(element: T | null) => void, boolean] {
   const isDisabled: boolean = disabled || !hasIntersectionObserver
 
   const unobserve = useRef<Function>()
   const [visible, setVisible] = useState(false)
-
+  const [myRoot, setMyroot] = useState(rootEl ? rootEl.current : null)
```

Comment:
Can we change `myRoot` to `root` and change `setMyroot` to `setRoot`?

Reply:
Done